nfs/pkg/volume/provision.go

```diff
@@ -450,7 +450,11 @@ func (p *nfsProvisioner) createDirectory(directory, gid string) error {
 	}
 
 	if gid != "none" {
-		groupID, _ := strconv.ParseUint(gid, 10, 64)
+		groupID, err := strconv.ParseUint(gid, 10, 64)
+		if err != nil {
+			os.RemoveAll(path)
+			return fmt.Errorf("chgrp failed with error: %v", err)
+		}
```

Comment:
Notes for reviewers:

If `gid` is invalid, e.g. `bar`, `strconv.ParseUint` returns 0 and an error. For root users, if we don't check errors, `chgrp 0 <path>` will succeed anyway, which IMHO is not expected.
I think travis-ci check passed in the past because `go test` is run by non-root user.